traffic_ops/app/lib/UI/Server.pm

```diff
@@ -980,7 +1018,14 @@ sub postupdate {
 
 	my $update_server =
 		$self->db->resultset('Server')->search( { id => $serverid } );
-	if ( defined $updated ) {
+
+	my $use_reval_pending = $self->db->resultset('Parameter')->search( { -and => [ 'name' => 'use_reval_pending', 'config_file' => 'global' ] } )->get_column('value')->single;
+
+	if ( defined($use_reval_pending) && $use_reval_pending == 1 ) {
```

Comment:
you can save a few lines of code here by getting rid of the else and moving 

$update_server->update( { upd_pending => $updated } );

below the remaining if...and also leave it out of the if...